packages/cli/src/commands/build.js

```diff
@@ -8,7 +8,6 @@ exports.build = async function buildCommand(src, argv) {
 	argv.src = src || argv.src;
 	// add `default:true`s, `--no-*` disables
 	argv.prerender = toBool(argv.prerender);
-	argv.production = toBool(argv.production);
```

Comment:
`production` is not a valid CLI flag, so it's been moved into `run-webpack.js` with the other derived env values.